src/components/HypothesisTesting/HTSimulation.js

```diff
@@ -20,6 +24,10 @@ export default function HTSimulation() {
 
   return (
     <div className="MainContainer">
+        <div>
```

Comment:
Maybe add this to `TestInputs.js` to keep all of those buttons in the same component?